src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good to see the use of `React.useRef()` for managing the BottomSheetMenu reference. However, it would be beneficial to add a comment explaining its purpose in this context.



<!-- This is an auto-generated comment by OSS CodeRabbit -->